e2e/app/page/google-login.ts

```diff
@@ -107,21 +119,18 @@ export default class GoogleLoginPage {
     const user = email || config.userEmail;
     const pwd = paswd || config.userPassword;
 
+    if (!user || user.trim().length === 0) {
```

Comment:
Why is this getting removed from the `try`?

Reply:
Problem if any usually happens after click Next button (after entering password).